calico_containers/pycalico/datastore.py

```diff
@@ -82,8 +82,12 @@
 DEFAULT_LOG_SEVERITY_SCREEN = "info"
 DEFAULT_LOG_FILE_PATH = "none"
 
-# Default global IP in IP.
-DEFAULT_IP_IN_IP = "true"
+# Default global IP in IP.  By default this is turned off and turned on as
```

Comment:
Wording is kind of awkward here, sounds like it is 'turned off and turned on'.
Maybe saying "This is off by default and is only turned on when required by an IP Pool."